index.html

```diff
@@ -14,41 +14,251 @@
+                </footer>
+            </article>
+            <article class="work-item">
+                <header>
+                    <figure class="work-img"><img src="assets/images/project-tonic-2.png"
+                            alt="Project Tonic 2 Preview Image"></figure>
+                    <h2>Tonic</h2>
+                    <div class="project-info">
+                        <div class="project-client">CANOPY</div>
+                        <div class="project-info-text"><i class="fa-solid fa-circle fa-2xs"></i></div>
+                        <div class="project-info-text">Back End Dev</div>
+                        <div class="project-info-text"><i class="fa-solid fa-circle fa-2xs"></i></div>
+                        <div class="project-info-text">2015</div>
+                    </div>
+                </header>
+                <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <ul class="work-categories">
+                    <li class="work-cat">html</li>
+                    <li class="work-cat">css</li>
+                    <li class="work-cat">javaScript</li>
+                </ul>
+                <footer>
+                    <div><button class="see-project-button">See Project</button></div>
+                </footer>
+            </article>
+            <article class="work-item">
+                <header>
+                    <figure class="work-img"><img src="assets/images/project-multi-post-2.png"
+                            alt="Project Multi Post 2 Preview Image"></figure>
+                    <h2>Multi-Post Stories</h2>
+                    <div class="project-info">
+                        <div class="project-client">CANOPY</div>
+                        <div class="project-info-text"><i class="fa-solid fa-circle fa-2xs"></i></div>
+                        <div class="project-info-text">Back End Dev</div>
+                        <div class="project-info-text"><i class="fa-solid fa-circle fa-2xs"></i></div>
+                        <div class="project-info-text">2015</div>
+                    </div>
+                </header>
+                <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <ul class="work-categories">
+                    <li class="work-cat">html</li>
+                    <li class="work-cat">css</li>
+                    <li class="work-cat">javaScript</li>
                 </ul>
-            </section>
-        </main>
-    </div>
+                <footer>
+                    <div><button class="see-project-button">See Project</button></div>
+                </footer>
+            </article>
+        </section>
+        <section class="about-myself">
+            <h1>About Myself</h1>
+            <p class="main-normal-text">Hello I’m a software developer! I can help you build a product , feature or
```

Comment:
- [ ] Please replace this `h1` with `h2` because each html page should contain only one h1 tag and to follow the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :blush: